content/en/news/security/istio-security-2020-008/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: ISTIO-SECURITY-2020-008
+subtitle: Security Bulletin
+description: Incorrect validation of wildcard DNS Subject Alternative Names.
+cves: [CVE-2020-15104]
+cvss: "6.6"
+vector: "AV:N/AC:H/PR:H/UI:N/S:C/C:H/I:L/A:N/E:F/RL:O/RC:C"
+releases: ["1.5.0 to 1.5.7", "1.6.0 to 1.6.4", "All releases prior to 1.5.0"]
+publishdate: 2020-07-09
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+Istio is vulnerable to a newly discovered vulnerability:
```

Comment:
```suggestion
A vulnerability affecting the TLS certificates subdomain validation of Envoy has been fixed and publicly disclosed (c.f. [tls: improve wildcard matching](https://github.com/envoyproxy/envoy/pull/11921) )
```

Reply:
I don't want to do this one.  I think it reads like we're just blaming Envoy and really it was Istio's choice to use Envoy so Istio is vulnerable.